src/os_kfbsd.h

```diff
@@ -0,0 +1,59 @@
+#ifndef FREEBSD_KERNEL_H
+#define FREEBSD_KERNEL_H
+
+double __floatdidf(long a) {
+    //todo: STELIOS
```

Comment:
@stelkasouridis : can you please emit a warning if these methods are called? I am worried that they are going to fail silently.